env/elasticsearch/Dockerfile

```diff
@@ -1,5 +1,5 @@
 #https://www.docker.elastic.co/
-FROM docker.elastic.co/elasticsearch/elasticsearch:6.5.2
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.6.1
```

Comment:
Could you open a separate PR with this change to make sure it does not have any other affects? With this change, we normally also update the README and all the places where we mention what we test with (just search for 6.5.2)

Reply:
Done in #1620

Reply:
Thanks, as soon as the other one is in, could you rebase on top of it?

Reply:
Sure, I will do it!